src/crimson/osd/osd.h

```diff
@@ -116,6 +118,8 @@ class OSD final : public crimson::net::Dispatcher,
   std::unique_ptr<Heartbeat> heartbeat;
   seastar::timer<seastar::lowres_clock> heartbeat_timer;
 
+  std::unique_ptr<crimson::admin::OsdAdmin>    osd_admin;
```

Comment:
there is no point to add the "osd" prefix. i think, all stuff in osd is related to "osd".

Reply:
OK.